Command/ImportCommand.php

```diff
@@ -111,14 +112,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
-     * Check if provided bundles namespace is correct.
+     * Check if provided bundles is correct.
      *
      * @param string $bundleName
+     * @param bool   $isShortName
      *
      * @throws InvalidArgumentException
      */
-    private function validateBundleNamespace($bundleName)
+    private function validateBundle($bundleName, $isShortName = false)
     {
+        if ($isShortName) {
+            $bundle = $this->getApplication()->getKernel()->getBundle($bundleName);
+
+            if (!$bundle) {
+                throw new InvalidArgumentException(
+                    "Invalid bundle '{$bundleName}'"
+                );
+            }
+
+            return 1;
```

Comment:
btw this return true should be at the end of function after all checks are completed. IMO not inside the if.